README.md

```diff
@@ -1,70 +1,197 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
```

Comment:
note: Since there is no open PR in the [backend repo](https://github.com/Tobby8629/Car_Rental_Backend) for me to review, I'll include my comments here for that project.

# Backend review

- [ ] IMPORTANT: The last 2 migrations are colliding against each other:

![image](https://github.com/Tobby8629/Car_rental_frontend/assets/57421823/77a26938-542b-4d55-a9a5-7df727b23ba7)

![image](https://github.com/Tobby8629/Car_rental_frontend/assets/57421823/fd2748d5-b7ab-4284-9ce5-1ad8ec3e7a92)

This is because they have the same name.

Kindly correct your migrations so your users can setup and run your project correctly.

_note: This means I am unable to run the server, and the frontend without it.  When this issue is fixed, the functionality and tests review will be continued._

- [ ] Kindly, remove the old commented code found in these files:

- cars_controller.rb
- reservations_controller.rb
- application_controller.rb
- models/car.rb

- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍

### -----End of backend review